applications/crossbar/priv/api/swagger.json

```diff
@@ -34475,19 +34475,44 @@
                     "description": "API key for Google Cloud Speech",
                     "type": "string"
                 },
+                "asr_enable_automatic_punctuation": {
+                    "default": "true",
```

Comment:
This should not be in quotes, I believe.

Reply:
No sure.. It was generated with make apis.. Other boolean values in my couchDB doc are in quote too.

Reply:
Ok well it failed the pipeline and `asr_use_enhanced` is a boolean without quotes that you added. But I'll leave it, just trying to help.